src/box/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ lua_source(lua_sources lua/load_cfg.lua load_cfg_lua)
 lua_source(lua_sources lua/schema.lua schema_lua)
 lua_source(lua_sources lua/tuple.lua tuple_lua)
 lua_source(lua_sources lua/session.lua session_lua)
+lua_source(lua_sources lua/iproto.lua iproto_lua)
```

Comment:
Is it really necessary to add a new Lua file to define the IPROTO constants? The constants could as well be exported from Lua C, just like we do with schema constants:

https://github.com/tarantool/tarantool/blob/d2d7f4de3a6ba2caa7f260860e750c733ee3201d/src/box/lua/space.cc#L741-L748

Reply:
Discussed with @locker: came to a conclusion it would be better to implement the whole `box.iproto` module in C — for instance, in scope of this patch, we can use enumeration constants directly instead of copy-pasting their values.